lib/mercadopago/client.rb

```diff
@@ -142,6 +144,27 @@ def search(search_hash)
     end
 
     #
+    # Performs a generic GET to the given URL
+    #
+    # - url: the URL to request
+    #
+    def get(url)
+      MercadoPago::Request.wrap_get("#{url}?access_token=#{@access_token}")
```

Comment:
@Miguelcldn I'm sorry, I didn't mean "payload", I meant "headers". 

Sure, it doesn't need a payload, but someone might want to send specific headers. 

See: https://github.com/ombulabs/mercadopago/blob/0276edda3e29336f797028317d6c6aec090e3740/lib/mercadopago/request.rb#L34-L36

Also, in Mercado Pago's API doc: https://www.mercadopago.com.ar/developers/en/api-docs/account/movements/

If we are going to add this feature, we should make sure that it's flexible enough to take headers like `accept: application/json`

Could you change that in the new `get` and `post` methods? Just add an optional parameter as the last parameter.